zerver/actions/user_topics.py

```diff
@@ -51,10 +87,10 @@ def do_unmute_topic(
     user_profile: UserProfile, stream: Stream, topic: str, *, skip_muted_topics_event: bool = False
 ) -> None:
     # Note: If you add any new code to this function, the
-    # remove_topic_mute call in do_update_message will need to be
+    # remove_topic_visibility_policy call in do_update_message will need to be
     # updated for correctness.
```

Comment:
I think this comment can be removed now, since do_update_message no longer makes such call.